README.md

```diff
@@ -3,8 +3,44 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. <br />
```

Comment:
dwyl will _always_ mean something. 
otherwise it's a _pointless_ "made up" word like "accenture" (_which stands for nothing_) 
Please, let's focus on what we need to _make_ not on whether the name is an acronym. We can all agree that there are _incredibly_ successful products/companies with _terrible_ names. "Joylent", "Starbucks", "Google"...?!
**users don't _care_** if it's an acronym, they care if the product _works_ and solves their pain/problem. Discussions around the name are a ***fruitless distraction*** that offers no benefit to anyone. If the company name has _meaning_ to the person who conceived it, that's good enough for me. ❤️  ✅ 
Our **_sole_ focus** for the next 6 months is to make sure we are cashflow positive (_i.e. not burn through our hard-earned capital_) not to argue about names or other details that _users_ don't care about.
If we think the name is _hindering_ us from being successful  (_please provide supporting evidence_) then we can discuss the options. 👍